examples/config-boltdb.json

```diff
@@ -0,0 +1,15 @@
+{
+    "distSpecVersion": "1.0.1-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
+        "dedupe": true,
+        "remoteCache": false
```

Comment:
this defaults to false if not specified i suppose?

Reply:
Yes